proto/streamlit/proto/Skeleton.proto

```diff
@@ -19,6 +19,15 @@ syntax = "proto3";
 option java_package = "com.snowflake.apps.streamlit";
 option java_outer_classname = "SkeletonProto";
 
-// An internal-only element that displays an app skeleton.
+// An empty-like element that displays an app skeleton.
 message Skeleton {
+  enum SkeletonStyle {
+    ELEMENT = 0;
+    APP = 1; // internal-only, for now
+  }
+  // Skeleton visual style
+  SkeletonStyle style = 1;
+
+  // Height in CSS points
```

Comment:
Nit: this is pixels, not points.

Reply:
oh, added it to this PR: https://github.com/streamlit/streamlit/pull/8508

Reply:
To be fair, I knew this was wrong when I wrote it but I went for consistency because I looked at other code in the repo and it all says "css points" for things that appear to be measured in pixels. It's totally possible i misread the other stuff tho.

Reply:
Hah, then we should fix those too at some point 😆 

Either way, thanks for the contribution @Asaurus1 !

Reply:
https://github.com/streamlit/streamlit/pull/8508 fixes those wrong "css points" comments as well 👍